user/customizing-the-build.md

```diff
@@ -49,13 +49,12 @@ $ travis settings maximum_number_of_builds --set 1
 
 ## Building only the latest commit
 
-If you are only interested in building the most recent commit on each branch you can use this new feature to automatically cancel older builds in the queue that are *not yet running*.
+If you are only interested in building the most recent commit on each branch you can use this new feature to automatically cancel older builds in the queue that are *not yet running*. Existing builds will be allowed to finish.
```

Comment:
```suggestion
If you are only interested in building the most recent commit on each branch you can use this new feature to automatically cancel older builds in the queue that are *not yet running*. Existing builds are allowed to finish.
```

Reply:
I'm sorry for not responding to this before. Today was when I got time to come back to all my outstanding contributions. I also wanted to add a comma to this and review if anything else needed corrections, but I guess it's okay as is.

Thanks!